data-prepper-plugins/mutate-event-processors/src/main/java/org/opensearch/dataprepper/plugins/processor/mutateevent/RenameKeyProcessorConfig.java

```diff
@@ -10,11 +10,13 @@
 import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 import com.fasterxml.jackson.annotation.JsonPropertyDescription;
 import jakarta.validation.Valid;
+import jakarta.validation.constraints.AssertTrue;
```

Comment:
Unused imports this line and line 19 below. You can try build locally and make sure it passes.

Reply:
Fixed it